certified-connectors/Morf/apiDefinition.swagger.json

```diff
@@ -0,0 +1,121 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Morf is a modern and lightweight API-driven platform that offers a rapid and flexible approach to form generation.",
+    "version": "1.0",
+    "title": "Morf",
+    "termsOfService": "https://github.com/aftialabs/morf-docs/blob/main/guides/termsandconditions.md",
+    "contact": {
+      "name": "Morf Support",
+      "url": "https://getmorf.io/support",
+      "email": "support@getmorf.io"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://getmorf.io"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://github.com/aftialabs/morf-docs/blob/main/guides/DataandPrivacyPolicy.md"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity;Sales and CRM"
+    }
+  ],
+  "host": "api.getmorf.io",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/": {
+      "post": {
+        "summary": "Render a new Morf data capture experience (V1)",
+        "description": "Call this API with a Morf form definition and optionally data to receive a prefilled form document that can be presented to a user in a browser context.",
+        "operationId": "render",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "render",
+            "schema": {
+              "$ref": "#/definitions/RenderRequest"
+            },
+            "description": "A form render request",
+            "required": true
+          }
+        ],
+        "security": [
+          {
+            "API-Key": []
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A form render response",
+            "schema": {
+              "$ref": "#/definitions/RenderResponse"
+            }
+          },
+          "401": {
+            "description": "Invalid Authentication"

```

Comment:
@GuillaumeCleme Empty or no response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
Added the schema object for responses and removed on invalid response @sriyen-msft